packages/@ember/controller/lib/controller_mixin.js

```diff
@@ -43,5 +46,13 @@ export default Mixin.create(ActionHandler, {
     @property model
     @public
   */
-  model: EMBER_METAL_TRACKED_PROPERTIES ? tracked() : null,
+  model: computed({
+    get() {
+      return this[MODEL];
```

Comment:
We may want to make this non-enumerable if there is an easy way to do it, but if it's too annoying it's not a big deal.

Reply:
The only way to do it currently is something I think we'd rather deprecate sooner rather than later (it requires some not-so-nice prototype chain manipulation), so I'd rather not for the moment